doc/releases/release-notes-2.1.rst

```diff
@@ -46,7 +46,46 @@ Bluetooth
 Build and Infrastructure
 ************************
 
-* TBD
+* Various parts of the binding format have been simplified. The format is
+  better documented now too.
+
+  This turns e.g.
```

Comment:
For clarity, it there a tool turning one yaml format into the other, or does the user need to write their yaml using the new format?  I think it's the latter, so change this to:

```
Instead of writing, for example, this description of an XXXX:

Reply:
Rephrased it to something similar. Does it look okay?

Reply:
(No tool btw.)